packages/back-end/src/models/FeatureModel.ts

```diff
@@ -59,6 +59,7 @@ const featureSchema = new mongoose.Schema({
           enabled: Boolean,
         },
       ],
+      applyScheduleRules: Boolean,
```

Comment:
this probably shouldn't be in here. I think we only want to persist the rules (despite the on/off state) within the modal itself, not the data model.